### PR TITLE
5 15

### PR DESCRIPTION
**Pull request by jasonLaster**

### CODE REVIEW

File: src/components/Editor/Editor.css [hunk 1]
```diff
@@ -1,8 +1,12 @@
-/* vim:set ts=2 sw=2 sts=2 et: */
+.editor-wrapper {
+  --debug-line-background: rgba(226, 236, 247, .5);
```
Comment (codehag) [inline src/components/Editor/Editor.css]: hmmm could these colors go into variables?

Comment (jasonLaster) [inline src/components/Editor/Editor.css]: this is defining a variable :)

Comment (codehag) [inline src/components/Editor/Editor.css]: ah COOOL

**Comment by codehag** [comment]
we have a small linting issue 
```
src/components/Editor/Editor.css
 2:48  ✖  Expected a leading zero   number-leading-zero
```

**Pull request merged**